lib/libutee/include/tee_bench.h

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2014, Linaro Limited
+ *
+ * This software is licensed under the terms of the GNU General Public
+ * License version 2, as published by the Free Software Foundation, and
+ * may be copied, distributed, and modified under those terms.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ */
+
+#ifndef TEE_BENCH_H
+#define TEE_BENCH_H
+
+#include <inttypes.h>
+
+#define UNUSED(x) (void)(x)
+
+/* max amount of timestamps */
+#define TEE_BENCH_MAX_STAMPS	10
```

Comment:
If it's enough for now, sure, then there is no reason to change it. It just sounded like a small number when thinking about all places where measurements can occur.